### PR TITLE
woocommerce integration

### PR DESCRIPTION
**Pull request by tyalau**

### CODE REVIEW

File: docs/integrations/custom/woocommerce.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# WooCommerce Integration
+
+###Setup your Metisa account
+Create your login name, team name, and team URL at https://askmetisa.com/create. You should receive an email with instructions to create a password and login.
+
+### Customise your theme templates
+You can safely customise the theme templates of your store using **overrides**. Details can be found [here](https://docs.woocommerce.com/document/template-structure/).
+
```
Comment (nickangtc) [inline docs/integrations/custom/woocommerce.md]: I suggest adding a short paragraph here to explain that they need to SSH into their file system in order to add and edit files (perhaps WC has their own article on this, just link it). Most people probably don't know how to go about doing it, so this is crucial

Comment (tszyanalau) [reply docs/integrations/custom/woocommerce.md]: May I know the link of the article? I didn't find it in their website.

**Comment by nickangtc** [comment]
Ready to merge. Deploy only when launching custom integrations! 🚀

File: docs/integrations/custom-ecommerce/woocommerce.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# WooCommerce Integration
+
+###Setup your Metisa account
```
Comment (jyek) [inline docs/integrations/custom-ecommerce/woocommerce.md]: add a space here

File: docs/SUMMARY.md [hunk 3]
```diff
@@ -7,6 +7,7 @@
   * [Magento](integrations/magento.md)
   * [BigCommerce](integrations/bigcommerce.md)
   * [Custom E-Commerce](integrations/custom-ecommerce.md)
+    * [WooCommerce](integrations/custom-ecommerce/woocommerce.md)
```
Comment (jyek) [inline docs/SUMMARY.md]: Put WooCommerce just below BigCommerce with the same indentation

**Comment by tyalau** [comment]
Im not sure whether to include the section for recommendation widget.

**Pull request merged**